src/app/components/outing/addOuting/index.js

```diff
@@ -39,13 +39,8 @@ class AddOuting extends Component {
 
   getWinner() {
     const { splurgingHard } = this.state;
-    let groupings = null;
+    let groupings = _(splurgingHard).groupBy(p => p.numCoffeeDrank / p.numCoffeePaid).value();
```

Comment:
I thought you were going to change it to do minus. This is just the same without the grouping which that accounts for division by 0. So this breaks if numCoffeePaid is 0.

You should test this locally before submitting a pull request :)